tesseract-server/src/app.rs

```diff
@@ -59,20 +64,32 @@ pub fn create_app(backend: Box<dyn Backend + Sync + Send>, db_type: Database, sc
         .resource("/cubes/{cube}", |r| {
             r.method(Method::GET).with(metadata_handler)
         })
+
+        // Aggregation
         .resource("/cubes/{cube}/aggregate", |r| {
             r.method(Method::GET).with(aggregate_default_handler)
         })
         .resource("/cubes/{cube}/aggregate.{format}", |r| {
             r.method(Method::GET).with(aggregate_handler)
         })
+
+        // Logic Layer
+        .resource("/logic-layer", |r| {
```

Comment:
I realized after looking at logic layer docs that they use something like `data` for the endpoint. So perhaps it's still worth bikeshedding. But no need to before this PR gets merged.